### PR TITLE
[feat] redis/redis cluster example

### PR DESCRIPTION
**Pull request by vernak2539**

### CODE REVIEW

**Comment by gergelyke** [comment]
looks great, thanks @vernak2539! can you make sure to run the linter against your example?

**Comment by vernak2539** [comment]
should be updated now

**Comment by gergelyke** [comment]
looks great, ty @vernak2539 !

File: example/redis/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM redis:latest
```
Comment (gshively11) [inline example/redis/Dockerfile]: I might be missing something obvious here, but how does the index.js actually get used in the image? Wouldn't the Dockerfile need to have the nodejs runtime and wire up the index.js as an ENTRYPOINT/CMD?

Comment (vernak2539) [reply example/redis/Dockerfile]: in this case, the docker file is there just to spin up the service the example is connecting to

Comment (gshively11) [reply example/redis/Dockerfile]: Ah, that makes sense, thanks

**Comment by vernak2539** [comment]
due to a travis timeout, seems the build failed and i don't have rights to start another one. If someone could kick it off, would be appreciated

**Comment by gergelyke** [comment]
just restarted

**Comment by vernak2539** [comment]
looks like travis is still having a lot of problems 😦

**Comment by gergelyke** [comment]
previously it passed - going to merge as is

**Pull request merged**